CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/AnalysisViewController.swift

```diff
@@ -66,10 +66,22 @@ import UIKit
     
     fileprivate lazy var loadingIndicatorText: UILabel = {
```

Comment:
@vizaknaidavid Could you use private instead of fileprivate whenever it's not necessary?